v7.h

```diff
@@ -23,7 +23,10 @@
 extern "C" {
 #endif /* __cplusplus */
 
+#include <stddef.h>   /* For size_t */
+
 #define V7_VERSION "1.0"
+#define V7_GENERIC_OBJECT_PROTOTYPE 0     /* For v7_create_object() */
```

Comment:
This is an valid `val_t` for double `+0`.

Why not use V7_NULL for NULL prototype and V7_UNDEFINED for default proto?